client/pages/view-entries.jsx

```diff
@@ -108,6 +111,28 @@ function Entries(props) {
     if (sharedEdit) setSharedEdit(false);
     if (!sharedEdit) setSharedEdit(true);
   }
+  function handleCommentSubmit(e) {
+    e.preventDefault();
+    const commentsObject = { postcomments };
+    fetch(`/api/comments/${entryId}`, { method: 'POST', body: JSON.stringify(commentsObject), headers: { 'Content-Type': 'application/json', 'x-access-token': window.localStorage.getItem('react-context-jwt') } })
+      .then(() => {
+        loadComments();
+      });
```

Comment:
It's not efficient to download the whole list again. Instead, concatenate the new comment into the list and set the comments to the updated list.